contrib/kafka/config/kafka.yaml

```diff
@@ -67,24 +67,6 @@ rules:
       - kafka-channel-dispatcher
     verbs:
       - update
-  - apiGroups:
-      - networking.istio.io
-    resources:
-      - virtualservices
-    verbs:
-      - get
-      - list
-      - watch
-      - create
-      - update
-  - apiGroups:
-      - "" # Core API Group.
```

Comment:
This got removed by mistake. Adding it back in some time after resolving merge conflicts with master

Reply:
Right, so the networking.istio.io / virtualservices is what I was proposing on removing. The core API group should not be removed.